.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "evm/basin_storage/lib/forge-std"]
 	path = evm/basin_storage/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "evm/basin_storage/lib/filecoin-solidity"]
-	path = evm/basin_storage/lib/filecoin-solidity
-	url = https://github.com/filecoin-project/filecoin-solidity
```

Comment:
removing `filecoin-solidity` deps since we are not working with deals at the contract level